examples/aws-atlas-privatelink-regionalized/atlas-cluster.tf

```
@@ -0,0 +1,47 @@
+resource "mongodbatlas_cluster" "cluster-atlas" {
+  project_id                   = var.atlasprojectid
+  name                         = "cluster-atlas"
+  cloud_backup                 = true
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.2"
```

```suggestion
  mongo_db_major_version       = "5.0"
```
A small thing but can you make this 5.0?  4.2 is pretty old for a new example.